dashboard/app/controllers/concerns/pd/workshop_filters.rb

```diff
@@ -0,0 +1,46 @@
+module Pd::WorkshopFilters
+  extend ActiveSupport::Concern
+
+  QUERY_BY_SCHEDULE = 'schedule'
+  QUERY_BY_END = 'end'
```

Comment:
these constants could use a brief description. whatever text you're showing in the UI to explain what they each are is probably good enough.